docs/misc/CTS_LUT_Discussion.md

```diff
@@ -58,6 +58,6 @@ set_input_transition
 on each input as needed, vary, rerun report_timing output load by
 set_load
 
-We can create a new tcl command in openroad.i to create a second opensta
+We can create a new Tcl command in openroad.i to create a second opensta
```

Comment:
Are these notes for future reference or instructions to the user? Resolve in next pass

Reply:
Will remove since this was kept only for history.